OG/OG/AST/MathExpression/MathExpressionVisitor.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using Antlr4.Runtime.Misc;
+using OG;
+using OG.AST;
+using OG.AST.Terminals;
+
+namespace OG.AST.MathExpression
+{
+    public class MathExpressionVisitor : OGBaseVisitor<NumberNode<int>>
```

Comment:
Denne NumberNode\<int> skal selvfølgelig gøres mere generisk, så vi også har mulighed for floats, decimal, mm.

Hvordan det skal gøres er dog en opgave til en anden gang :)

Reply:
enig